llvm/lib/Target/AArch64/AArch64FrameLowering.cpp

```diff
@@ -3334,8 +3334,9 @@ static void computeCalleeSaveRegisterPairs(
         break;
       case RegPairInfo::FPR64:
         if (AArch64::FPR64RegClass.contains(NextReg) &&
-            !invalidateWindowsRegisterPairing(RPI.Reg1, NextReg, NeedsWinCFI,
-                                              IsFirst, TRI))
+            !invalidateRegisterPairing(
+                RPI.Reg1, NextReg, IsWindows, NeedsWinCFI,
+                /*NeedsFrameRecord=*/false, IsFirst, TRI))
```

Comment:
Is there a reason to change this to `invalidateRegisterPairing`? IIUC the only difference is the `NeedsFrameRecord` case (which is set to false here).